### PR TITLE
finish mobile version project

### PR DESCRIPTION
**Pull request by DamenaM**
# Request of  PR I  finish the mobile version project

- [ ] create Works section
- [ ] check  linter error
- [ ] Create About myself section
- [ ]  adding cards template using Flexbox to lay out 
- [ ] use CSS Grid and Flexbox

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -101,3 +100,235 @@ body {
   display: flex;
   gap: 20px;
 }
+
+/* social media icons----------------------------------------------------------------------- */
+.Workssection {
+  border-radius: 8px;
+}
```
Comment (Olamarx) [inline style.css:107]: - [ ] For proper organizing of the content in this tag, could you consider styling it with grid as display property? You can read about the usefulness [here](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns)

File: index.html [hunk 2]
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
                 <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
             </ul>
             </div>
-
 </section>
 </section>
+<section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (1).png" alt=""></a>
```
Comment (Olamarx) [inline index.html:35]: - [ ] Kindly consider using this picture through CSS as a background image as decorations are always backgrounded images, not img tags.

File: index.html [hunk 3]
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
...
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
```
Comment (Olamarx) [inline index.html:51]: - [ ] Kindly consider using ul tag and each takes li tag to satisfy semantic purpose.

File: index.html [hunk 4]
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
...
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
```
Comment (Olamarx) [inline index.html:53]: - [ ] Kindly consider the color as used on the figma
### Figma
image

### Your Design
image

File: index.html [hunk 5]
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
...
+  </section>
+  <section class="Workssection">
+    <div class="cardBack2">
+        <h3  class="tonic" >About Myself </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
```
Comment (Olamarx) [inline index.html:145]: - [ ] This appears not to be on the Figma design. I would recommend sticking to the Figma template.

### Your Design
image

### Figma Design
image

File: index.html [hunk 6]
```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
...
+    <p class="section1"> Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me. </p>
+</div>
+<div class = "social-section">
+    <h4 class="connectmywork"> LET’S CONNECT</p>
```
Comment (Olamarx) [inline index.html:151]: - [ ] This confuses me. It is opening the tag with h4 and classes it with p tag 😒.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @DamenaM, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✔️ 
- Good commit messages ✅ 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please consider applying the comment about button to all the buttons.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DamenaM** [comment]
> based on PR comment

- [ ] border-radius
- [ ] I have changed my background image to CSS 
- [ ] I have changed div tag ul tag 
- [ ] I Have changed button color to figma design

File: index.html [hunk 7]
```diff
@@ -8,29 +8,173 @@
...
+        <ul class="Program-skil">
+            <li  class="Programs-lang" href="#"><img  src="./image/Frame 3.png" alt=""></li>
+            <ul class="Program-skil">
+                <li  class="Programs" href="#"><img src="./image/Frame 9.png" alt=""></li>
+                  <li  class="Programs" href="#"><img src="./image/Frame 10.png" alt=""></li>
+                  <li  class="Programs" href="#"><img src="./image/Frame 11.png" alt=""></li>
+              </ul>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 01.png" alt=""></li>
+            <li  class="Programs-lang" href="#"><img src="./image/Frame 02.png" alt=""></li>
+        </ul>
```
Comment (sajid-munawar) [inline index.html:175]: - [Optional] Iam sorry that it was not pointed out by the previous reviewer, I strongly suggest you to change the languages from image to text as it is not an image. they are texts.
img

**Review by sajid-munawar — CHANGES_REQUESTED** [review]
Hi @DamenaM , 

_Great job so far 👏 💯 👍. You've done well implementing some of the requested changes. However, there are still some which aren't addressed yet. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 
  - Addressed some of the issues raised from the previous review. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
  
  - N/A 

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**